### PR TITLE
Use 'center' tex_environment instead of \centering

### PR DESCRIPTION
**Pull request by cobordism**
See https://github.com/ManimCommunity/manim/issues/556

Tl;DR 
Main change:
remove `alignment="\\centering"` in favour of `tex_environment="center"`

Further changes:
- `tex_environment` can now take more complex tex environments like "{tabular}[t]{crlr}" and not just "tabular".
- small fix when attempting to manipulate a `TexTemplateFromFile` (it now crashes for the right reason)

Interesting to note:
`Tex("abcde")` is now the exact same thing as `MathTex("abcde", tex_environment="center")`
`Tex("abcde", tex_environment="align*")` is now the exact same thing as `MathTex("abcde")`
...but there is still a small technical difference when using multiple strings in the arguments.

### CODE REVIEW

**Comment by cobordism** [comment]
note: the check failure is a network timeout in installing manim, not an error in this PR

**Review by PhilippImhof — APPROVED** [review]
Nice improvement. Looks good to me and works fine with different environments. I also checked that thanks to the `standalone` class `\centering Bla` and `\begin{center}Bla\end{center}` yield the same SVG, even though in normal documents, the result would not be identical due to different vertical spacing.

In `_texcode_for_environment` the new `removeprefix` and `removesuffix` methods from https://www.python.org/dev/peps/pep-0616/ might come in handy in the future, but they are just being introduced.

**Review by eulertour — APPROVED** [review]

File: docs/source/changelog.rst [hunk 1]
```diff
@@ -68,6 +68,7 @@ Mobjects, Scenes, and Animations
 #. :code:`Scene` now renders when :code:`Scene.render()` is called rather than upon instantiation.
 #. :code:`ValueTracker` now supports increment using the `+=` operator (in addition to the already existing `increment_value` method)
 #. Add :class:`PangoText` for rendering texts using Pango.
+#. The ``alignment`` attribute to Tex and MathTex has been removed in favour of ``tex_environment``.
```
Comment (behackl) [inline docs/source/changelog.rst:71]: We should introduce a new section "next release" or so above the v0.1 section in the `changelog.rst`, and this line should be moved there.

File: manim/utils/tex.py [hunk 2]
```diff
@@ -169,6 +171,44 @@ def get_texcode_for_expression(self, expression):
...
+
+        Parameters
+        ----------
+        environment : :class:`str`
+            The tex_environment as a string. Acceptable formats include:
+            ``{align*}``, ``align*``, ``{tabular}[t]{cccl}``, ``tabular}{cccl``, ``\\begin{tabular}[t]{cccl}``.
+
+        Returns
+        -------
+        :class:`str` :class:`str`
```
Comment (behackl) [inline manim/utils/tex.py:186]:
```suggestion
        Tuple[:class:`str`, :class:`str`]
```

**Review by behackl — CHANGES_REQUESTED** [review]
LGTM, thanks for further cleaning this up!

Please move the line in the changelog and consider the suggestion I made for the docstring. :-)

File: docs/source/changelog.rst (hunk 1 above)
Comment (cobordism) [reply docs/source/changelog.rst]: I added a new section for 0.2.0 ("changes since v0.1.0")

**Review by behackl — APPROVED** [review]
